### DISCUSSION TITLE
hidden property not working

### DISCUSSION THREAD

**Discussion by KKaranja** [tailwindlabs/tailwindcss#9637] [Help]
Hi, I have been having a problem with the hidden property, when I use it to hide an element in small devices but show them in large devices it doesn't show them even on large screens, what could be the issue?

here is my code example:

<div className='hover-effect flex items-center text-gray-700 justify-center xl:justify-start text-lg space-x-3'>
      <Icon className='h-7' />
      <span className={`${active && "font-bold"} hidden xl:inline`}>
        {text}
      </span>
    </div>
  );

**Comment by wongjn** [comment]
It does not seem the code example has actually been provided. Consider creating a [Tailwind Play](https://play.tailwindcss.com) that demonstrates your issue.
